source/TR-181/middle_layer_src/wanmgr_rdkbus_apis.h

```diff
@@ -61,6 +61,7 @@ struct IFACE_INFO
     CHAR    ActiveStatus[BUFLEN_64];
     CHAR    CurrentActive[BUFLEN_64];
     CHAR    CurrentStandby[BUFLEN_64];
+    CHAR    CurrentActiveDNS[BUFLEN_256];
```

Comment:
IFACE_INFO linked list was initially added to handle multiple interfaces AvailableStatus in a sorted order. 
CurrentActiveDNS will be needed only for an active interface. Should we require it in the linked list?